collection-scripts/gather_nodes

```diff
@@ -104,8 +99,15 @@ do
         oc cp $pod:/host/etc/pcidp/config.json $NODE_PATH/pcidp_config.json -n node-gather 2>/dev/null
     fi
 
+}
+
+for line in $(oc get pod -o=custom-columns=NODE:.spec.nodeName,NAME:.metadata.name --no-headers --field-selector=status.phase=Running -n node-gather)
+do
+    gather_single_pod "$line" &
```

Comment:
wouldn't it create the job number per each call to the output ?

Reply:
Same - http://git.savannah.gnu.org/cgit/bash.git/tree/execute_cmd.c#n539

Try it with a script:
```
foo() {
    sleep 5 
}

foo &

echo waiting; wait; echo done
```
You'd get:

```
$ bash t.sh 
waiting
done
$
```